src/v1t/models/readout/readout.py

```diff
@@ -81,5 +81,5 @@ def __init__(
     def regularizer(self, mouse_id: int, reduction: str = "sum"):
         return self[str(mouse_id)].regularizer(reduction=reduction)
 
-    def forward(self, inputs: torch.Tensor, mouse_id: str, shifts: torch.Tensor = None):
-        return self[mouse_id](inputs, shifts=shifts)
+    def forward(self, inputs: torch.Tensor, mouse_id: str, query_neurons: torch.Tensor = None, shifts: torch.Tensor = None):
```

Comment:
same here `from typing import Optional` and 
```suggestion
    def forward(self, inputs: torch.Tensor, mouse_id: str, query_neurons: Optional [torch.Tensor] = None, shifts: Optional[torch.Tensor] = None):
```